docker/Dockerfile.ilogtail-toolchain-linux

```diff
@@ -65,6 +65,25 @@ RUN yum -y install gcc gcc-c++ make libuuid-devel libstdc++-static systemd-devel
 RUN [[ ${TARGETPLATFORM##*/} == 'amd64' ]] && ARCH='x86_64' || ARCH='aarch64' && \
     debuginfo-install -y glibc-2.17-326.el7_9.${ARCH} libuuid-2.23.2-65.el7_9.1.${ARCH}
 
+# install dependencies
+RUN yum -y install epel-release centos-release-scl
```

Comment:
please merge line 69 and 70 into one and append the following code to reduce the image size.
``` shell
&& yum -y clean all && rm -fr /var/cache && rm -rf /core.*
```